server/app.mjs

```diff
@@ -121,6 +121,24 @@ export const startAppServer = ({ dev, hostname, port, appDir }) => {
   });
 };
 
+function appSettingsLocation() {
```

Comment:
Agonizes me to have to put this in a third location but its necessary until we change the repo structure.

Reply:
Nick actually implemented the logic to initiliaze env variables here 
https://github.com/gptscript-ai/desktop/blob/71377ca94103170f598a9e5904daf1be7383f173/electron/config.mjs#L50

So it might be better to move the code there and just reference env variables in app.mjs

Reply:
Oh I see you already did this..